### ISSUE TITLE
Trending endpoint only returns videos owned by @TikTok

### ISSUE THREAD

**Issue by getsec** [davidteather/TikTok-Api#665]
First off, awesome lib.

Followed README and youtube tutorial, however, seems like this script isn't loading the trending page correctly (possible pebcak).

I've reset my `custom_verifyFp` multiple time, however, each time I call the `by_trending` method, the only responses I get are owned by @tiktok. I've also tried setting a few different opts as named params in the get_instance method, but still to no avail.

Examples below. Cheers!

# Request
```py
code = "verify_ksaqrte0_xxxxxxxxxxxxxxxxxxx"
api = TikTokApi.get_instance(custom_verifyFp=code)
x = api.by_trending(custom_device_id="xxx")
for i in x:
    print(i["author"]["uniqueId"])
```

# Response
```sh
python main.py
tiktok
tiktok
tiktok
tiktok
...
```

**Comment by lionneclement** [comment]
Same

**Comment by MuhammedEminK** [comment]
same

**Comment by n0ise9914** [comment]
same

**Comment by reccardt** [comment]
Me too.

**Comment by Scrip7** [comment]
same I guess

**Comment by getsec** [comment]
Guys dont comment on the PR if you have nothing to add. lol You raise my hopes that this gets an answer not 'same'

**Comment by sszczurek** [comment]
Proposal for the fix: https://github.com/sszczurek/TikTok-Api/commit/396cd84d5c59620747d14a5d2f2cb7844a2e6922. Unfortunately, I won't be able to open a PR here.

**Comment by isaackogan** [comment]
> Proposal for the fix: [sszczurek@396cd84](https://github.com/sszczurek/TikTok-Api/commit/396cd84d5c59620747d14a5d2f2cb7844a2e6922). Unfortunately, I won't be able to open a PR here.
> Also, this result from TikTok is like a "shadow ban" imho, meaning if the request is invalid or looks like scraping, TikTok returns only posts published by the offical account, pretending these are the trending posts. If you have a moment, we could add detection for such results, like if all posts are published by TikTok, raise an error.

Has anyone PR'd this yet?

**Comment by ejxtreme512** [comment]
any updates

**Comment by gilmarGNJ** [comment]
> Proposal for the fix: [sszczurek@396cd84](https://github.com/sszczurek/TikTok-Api/commit/396cd84d5c59620747d14a5d2f2cb7844a2e6922). Unfortunately, I won't be able to open a PR here.
> Also, this result from TikTok is like a "shadow ban" imho, meaning if the request is invalid or looks like scraping, TikTok returns only posts published by the offical account, pretending these are the trending posts. If you have a moment, we could add detection for such results, like if all posts are published by TikTok, raise an error.

ty

**Comment by ansoncmh** [comment]
same result, any updates?

**Comment by davidteather** [comment]
Seems to be fixed in V4.1.0
My code
```py
from TikTokApi import TikTokApi
api = TikTokApi.get_instance()
t = api.by_trending()

for v in t:
    print(v["author"]["uniqueId"])
```

Yields
```
kikakiim
bts_official_bighit
charlidamelio
kallmekris
jackblack
lalacataa
mikmikayy_
charlidamelio
willsmith
7seven777
charlidamelio
kikakiim
cheechandchong
miakhalifa
kikakiim
charlidamelio
charlidamelio
re...08
bdylanhollis
missmarymmouser
kikakiim
charlidamelio
kikakiim
mamalindy
lanie_barnett
my_vanh
jerryquinn870
my_vanh
charlidamelio
nikkietutorials
```
